config/docker/cros-baseline/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:bullseye-slim
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    apt-transport-https \
+    ca-certificates 
+
+RUN apt-get update && apt-get install -y \
+    git \
+    inetutils-ping \
+    ssh
+
+RUN \
+  mkdir -p /home/cros && \
+  useradd cros -d /home/cros && \
+  chown cros: -R /home/cros && \
+  adduser cros sudo && \
+  echo "cros ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+USER cros
+ENV HOME=/home/cros
+WORKDIR $HOME
+
+# All this just to extract rsa key to access DUT
+# Adding private key "as is" might trigger github audit alert
+RUN mkdir -p $HOME/trunk
+RUN git clone \
```

Comment:
Guess it could have been a shallow clone?

Reply:
I think yes, i will test it and if shallow clone works, will send update as it will make things faster